client/app/iplb/sslCertificate/iplb-ssl-certificate-edit.html

```diff
@@ -1,67 +1,51 @@
-                        id="key"
-                        name="key"
-                        required></textarea>
-                </div>
+        <oui-field label="{{'iplb_ssl_edit_private_key' | translate }}">
+            <div class="oui-field-control oui-field-control_5">
+                <textarea class="oui-textarea"
+                          data-ng-model="ctrl.ssl.key"
+                          id="key"
+                          name="key"
+                          required></textarea>
             </div>
-        </div>
+        </oui-field>
 
-        <div class="oui-field">
-            <label for="certificate" class="oui-label" data-translate="iplb_ssl_edit_certificate"></label>
-            <div class="oui-field__content">
-                <div class="oui-field-control oui-field-control_5">
-                    <textarea class="oui-textarea"
-                        data-ng-class="{ 'oui-textarea_error': ctrl.form.$submitted && ctrl.form.certificate.$invalid }"
-                        data-ng-model="ctrl.ssl.certificate"
-                        id="certificate"
-                        name="certificate"
-                        required></textarea>
-                </div>
+        <oui-field label="{{'iplb_ssl_edit_certificate' | translate }}">
+            <div class="oui-field-control oui-field-control_5">
+                <textarea class="oui-textarea"
+                          data-ng-model="ctrl.ssl.certificate"
+                          id="certificate"
+                          name="certificate"
+                          required></textarea>
             </div>
-        </div>
+        </oui-field>
 
-        <div class="oui-field">
-            <label for="chain" class="oui-label" data-translate="iplb_ssl_edit_chain"></label>
-            <div class="oui-field__content">
-                <div class="oui-field-control oui-field-control_5">
-                    <textarea class="oui-textarea"
-                        data-ng-model="ctrl.ssl.chain"
-                        id="chain"
-                        name="chain"></textarea>
-                </div>
+        <oui-field label="{{'iplb_ssl_edit_chain' | translate }}">
+            <div class="oui-field-control oui-field-control_5">
+                <textarea class="oui-textarea"
+                          data-ng-model="ctrl.ssl.chain"
+                          id="chain"
+                          name="chain"></textarea>
             </div>
-        </div>
+        </oui-field>
 
-        <div class="cui-form-actions">
-          <button type="submit"
-            class="oui-button oui-button_primary"
-            data-translate="iplb_ssl_add_confirm"
-            data-ng-click="ctrl.create()"></button>
-          <a class="oui-button oui-button_link"
-              data-translate="iplb_ssl_add_cancel"
-              data-ui-sref="network.iplb.detail.ssl-certificate"></a>
-        </div>
+        <oui-form-actions data-on-submit="ctrl.create()"
```

Comment:
Action should be on the form with ng-submit, not on the button. 
Buttons don't have type(reset|submit|button) in oui-form-actions, they should have one, it's a requirement.

Reply:
This is a problem. 👍

Reply:
Solved with ovh-ui-angular 2.5.3.